libcraft/items/src/inventory_slot.rs

```diff
@@ -13,33 +13,35 @@ pub enum InventorySlot {
 
 impl Default for InventorySlot {
     fn default() -> Self {
-        InventorySlot::Empty
+        Self::Empty
     }
 }
 
 impl From<Option<ItemStack>> for InventorySlot {
     fn from(it: Option<ItemStack>) -> Self {
         match it {
-            Some(item) => InventorySlot::Filled(item),
-            None => InventorySlot::Empty,
+            Some(item) => Self::Filled(item),
```

Comment:
Can replace this with a simple map and unwrap default. ie `it.map(Self::Filled).unwrap_or_default()`.